README.md

```diff
@@ -119,24 +119,25 @@ If the problem still exists search the following list of issues for possible sol
 
  <details>
     <summary><a href="https://www.tracetronic.com/products/ecu-test">ECU-TEST</a>/
-    <a href="https://www.tracetronic.com/products/trace-check/">TRACE-CHECK</a> 2021.4.1 or higher </summary>
+    <a href="https://www.tracetronic.com/products/trace-check/">TRACE-CHECK</a> 2022.4 or higher </summary>
```

Comment:
The current TRACE-CHECK version is 2023.3, the ticket only asks for ECU-TEST to be updated, but I think compatibility with TRACE-CHECK should also be checked. => `2023.3`

There are many TRACE-CHECK versions between these versions. Are they checked too?